packaging/pom.xml

```diff
@@ -111,6 +111,69 @@
                             </dataSet>
                         </configuration>
                     </plugin>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>rpm-maven-plugin</artifactId>
+                        <version>2.2.0</version>
+                        <executions>
+                            <execution>
+                                <id>generate-rpm</id>
+                                <goals>
+                                    <goal>rpm</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                        <configuration>
+                            <license>(c) Alibaba, GPLv3</license>
+                            <distribution>Arthas</distribution>
+                            <group>Development/tools</group>
+                            <packager>Alibaba</packager>
+                            <prefix>/usr/share</prefix>
+                            <defineStatements>
+                                <defineStatement>_unpackaged_files_terminate_build 0</defineStatement>
+                            </defineStatements>
+                            <mappings>
+                                <mapping>
+                                    <directory>/usr/share/arthas</directory>
+                                    <filemode>755</filemode>
+                                    <username>root</username>
+                                    <groupname>root</groupname>
+                                    <sources>
+                                        <source>
+                                            <location>${project.build.directory}/arthas-${project.version}-bin</location>
+                                        </source>
+                                    </sources>
+                                </mapping>
+                                <mapping>
+                                    <directory>/usr/local/bin</directory>
+                                    <filemode>755</filemode>
+                                    <username>root</username>
+                                    <groupname>root</groupname>
+                                    <sources>
+                                        <source>
+                                            <location>${project.build.directory}/arthas-${project.version}-bin/as.sh</location>
```

Comment:
`/usr/local/bin/as.sh` need to link `/usr/share/arthas/a.sh`, otherwise `as.sh` can not find the jars, it will download from remote maven repository.

Reply:
can you please check it now, also I have tested it in centos,  it's working fine